apps/ledger-live-mobile/ios/Podfile

```diff
@@ -52,6 +52,12 @@ target 'ledgerlivemobile' do
     # Pods for testing
   end
 
+  
+  # Double check if we need to be overriding this here
+  pod 'Starscream', '~> 4.0.0'
+  pod 'BleTransport', :git => 'https://github.com/LedgerHQ/hw-transport-ios-ble.git', :commit => '85e44468fff08ad60f77938b31b782f899b5d630'
```

Comment:
The native transport is written by @DantePuglisi and he will take care of pointing to an official release eventually. As of right now, pointing to a specific commit hash is good enough since new commits won't break our builds. Indeed an official release would be better. @DantePuglisi is going to update the code to match the latest API since it has diverged a bit so I'll let him take care of that too.